physionet-django/user/models.py

```diff
@@ -1218,8 +1218,7 @@ class CloudInformation(models.Model):
                                 on_delete=models.CASCADE)
     gcp_email = models.OneToOneField('user.AssociatedEmail', related_name='gcp_email',
                                      on_delete=models.SET_NULL, null=True)
-    aws_id = models.CharField(max_length=60, null=True, blank=True, default=None)
-
+    aws_id = models.CharField(max_length=60, null=True, blank=True, default=None, validators=[validators.validate_aws_id])
```

Comment:
I think you've lost the line before `class Meta`? It needs to be added back.

Reply:
@tompollard I accidentally did. Thanks for catching that!